### ISSUE TITLE
Assertion failed 'next != def && "found def after use"' in 'JIT.HardwareIntrinsics.General._Vector128_1.VectorGetAndWithElement__GetAndWithElementInt321:RunReflectionScenario(

### ISSUE THREAD

**Issue by EgorBo** [dotnet/runtime#93300]
Failed in `runtime-coreclr jitstress-random` pipeline
```
Beginning scenario: RunBasicScenario

Assert failure(PID 23 [0x00000017], Thread: 23 [0x0017]): Assertion failed 'next != def && "found def after use"' in 'JIT.HardwareIntrinsics.General._Vector128_1.VectorGetAndWithElement__GetAndWithElementInt321:RunReflectionScenario(int,ubyte):this' during 'Lowering nodeinfo' (IL size 537; hash 0x237e1f58; FullOpts)

    File: /__w/1/s/src/coreclr/jit/lir.cpp Line: 1665
    Image: /root/helix/work/correlation/corerun
```

https://dev.azure.com/dnceng-public/public/_build/results?buildId=431331&view=ms.vss-test-web.build-test-results-tab&runId=9538558&resultId=118570&paneView=dotnet-dnceng.dnceng-build-release-tasks.helix-test-information-tab

**Comment by jakobbotsch** [comment]
Very likely exposed by #92768, might already be fixed by #93210, I'll take a look.

```scala
export DOTNET_TieredCompilation=0
export DOTNET_JitStress=148
```

**Comment by jakobbotsch** [comment]
Tested the helix bits and overlaying a new libclrjit.so fixes the issue (while I could reproduce it before doing that), so indeed it looks to be fixed by #93210 already.
